tests/destructor/tconsume_twice.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   cmd: "nim c --newruntime $file"
+  disabled: "true"
```

Comment:
This test is valuable. You can create a custom type with an `.error` assignment operator to replicate `owned`'s behavior.